src/kibana/components/doc_table/doc_table.less

```diff
@@ -7,6 +7,12 @@ doc-table {
   margin: 5px;
   .flex(1, 1, 100%);
 
+  tr.discover-table-row  {
+    td:not(.discover-table-timefield):not(.ng-scope) {
```

Comment:
Then it gets other `td`s too - such as the one showing the timestamp and the other one showing the expand arrow.